_rules/line-height-not-important-78fd32.md

```diff
@@ -216,37 +219,37 @@ This `p` element has a [computed][] `line-height` of `19.2px` (120% of `16px`) w
 	}
 </style>
 
-<p style="line-height: 120% !important">
+<p style="line-height: 120% !important; max-width: 200px;">
 	The toy brought back fond memories of being lost in the rain forest.
 </p>
 ```
 
 #### Failed Example 4
 
-This `p` element has a [computed][] `line-height` of `1.2` which is below the recommended minimum.
+This `p` element has a [computed][] `line-height` of `1.2` which is below the required minimum.
 
 ```html
-<p style="line-height: 1.2 !important">
+<p style="line-height: 1.2 !important; max-width: 200px;">
 	The toy brought back fond memories of being lost in the rain forest.
 </p>
 ```
 
 #### Failed Example 5
 
-This `p` element has a [computed][] `line-height` of `normal` which is below the recommended minimum ([used][] value is generally around 1.2).
+This `p` element has a [computed][] `line-height` of `normal` which is below the required minimum ([used][] value is generally around 1.2).
```

Comment:
```suggestion
This `p` element has a [used][] `line-height` of `normal` which is below the required minimum ([used][] value is generally around 1.2).
```

Reply:
I think I got most of the others, but I don't think this should be changed. Wouldn't `normal` be the `computed` value and 1.2 be the `used` value as indicated by the comment at the end "([used][] value is generally around 1.2)".

Reply:
I agree 👍